projects/js-packages/charts/src/charts/geo-chart/geo-chart.module.scss

```diff
@@ -0,0 +1,6 @@
+.container {
+	position: relative;
+	display: flex;
+	justify-content: center;
+	align-items: center;
```

Comment:
I'd like to use `@wordpress/ui` `Stack` for this but there isn't a proper release including it ready yet: https://www.npmjs.com/package/@wordpress/ui?activeTab=versions